reference/dic_tags.rst

```diff
@@ -689,6 +689,10 @@ channel when injecting the logger in a service.
             ->addArgument(new Reference('logger'))
             ->addTag('monolog.logger', array('channel' => 'acme'));
 
+.. caution::
+    Even if you use `autowire` for your services, you need to inject the
+    `logger` service manually in this case.
```

Comment:
I'm not sure if this is still true. I can't confirm this for 3.3, but in 3.4 it's no longer needed. If this gets merged, it should be removed from 3.4 again.

The monolog bundle adds an alias:
```xml
<service id="Psr\Log\LoggerInterface" alias="logger" public="false" />
```

Reply:
The alias is for the "main logger". Currently, if you add the `monolog.logger` tag to get the logger for a particular channel, this will indeed not work as intended (see symfony/monolog-bundle#218).

However, I suggest to fix this in MonologBundle instead.

Reply:
see symfony/monolog-bundle#251